packages/ses/src/lockdown.js

```diff
@@ -218,17 +243,21 @@ export const repairIntrinsics = (options = {}) => {
   const { warn } = reporter;
 
   if (dateTaming !== undefined) {
-    // eslint-disable-next-line no-console
     warn(
       `SES The 'dateTaming' option is deprecated and does nothing. In the future specifying it will be an error.`,
     );
   }
   if (mathTaming !== undefined) {
-    // eslint-disable-next-line no-console
     warn(
       `SES The 'mathTaming' option is deprecated and does nothing. In the future specifying it will be an error.`,
     );
   }
+  if (hostEvaluators === '_legacy') {
+    // TODO: Remove when 'all' introduced as the new default option (breaking change).
+    warn(
+      `SES Please now use the 'hostEvaluators' option. In the future not specifying 'none' will error under strict CSP.`,
```

Comment:
possibly worth mentioning since 'all' will be the new default, but currently straight to the point